### DISCUSSION TITLE
There're no nodes connected

### DISCUSSION THREAD

**Discussion by LostPoE** [evilsocket/opensnitch#981] [General]
Just finished a fresh install of EndevourOS/Archlinux. Installed opensnitch with the Git AUR. When configuring the opensnitch app in the nodes section it says "There're no nodes connected".  The address reads "unix:///tmp/osui.sock" when trying to change this to a different address it does not save.

Any Ideas on how I can get opensnitch working?

![image](https://github.com/evilsocket/opensnitch/assets/3904973/a7750faf-9e04-4d02-947e-668ec31166ce)

Thanks

**Comment by gustavo-iniguez-goya** [comment]
Hi @LostPoE ,

What address did you try to use? See if the address of the daemon is updated in `/etc/opensnitchd/default-config-json`

**Comment by LostPoE** [comment]
unix:/local
![image](https://github.com/evilsocket/opensnitch/assets/3904973/526f0e24-18a6-4c02-8580-4a950a7aa811)

Edited. Manually updated  /etc/opensnitchd/default-config-json with unix:/local

However, the setting does not reflect whats in the UI.
![image](https://github.com/evilsocket/opensnitch/assets/3904973/f1f0d97b-dce0-4c41-bf00-a33a67fb3769)

**Reply by gustavo-iniguez-goya** [reply]
mmmh, interesting. 

> When configuring the opensnitch app in the nodes section (...)

Did you enter `unix:/local` manually on the GUI? Or what did you do exactly?

See what was saved to disk: `$ grep server_address ~/.config/opensnitch/settings.conf`

**Comment by LostPoE** [comment]
When I first installed /etc/opensnitchd/default-config-json had unix:///tmp/osui.sock. 
I then modified /etc/opensnitchd/default-config-json with unix:/local.
Restarted
Opened OpenSnitch and the unix:/local setting is not in the UI, like below.
![image](https://github.com/evilsocket/opensnitch/assets/3904973/ee5de499-434b-44df-9498-1f05b0a04a33)

**Reply by gustavo-iniguez-goya** [reply]
oh, I see. That's not supported, the daemon doesn't send manual changes to the GUI. Instead, you change the address from the GUI, it  applies the configuration to the daemon, and it suggests you to restart the GUI.

You could change the GUI address in `~/.config/opensnitch/settings.conf` (server_address).

On the other hand, `unix:/local` is not a valid unix socket address. If you set that address the GUI fill fail with the following error: `Failed to bind to address unix:/local`

If you want to change the path you can set it to `unix:///run/user/1000/opensnitch/osui.sock` , or another path where the GUI can write to.

**Comment by LostPoE** [comment]
I changed it to "unix:///run/user/1000/opensnitch/osui.sock" clicked apply and save and got this prompt. Restarted GUI. However, it still cant connect.
![image](https://github.com/evilsocket/opensnitch/assets/3904973/f82dcc0e-43b5-4bda-b231-c042f1731b46)

**Reply by gustavo-iniguez-goya** [reply]
Ok, please follow these instructions:
- Close the GUI.
- Change the value of LogLevel in `/etc/opensnitchd/default-config.json` to 0 ("LogLevel": 0)
- Restart the daemon: `sudo service opensnitch restart`
- Launch the GUI from a terminal as follow: `$ opensnitch-ui > /tmp/gui.log`

And post the following information:

- content of `/etc/opensnitchd/default-config.json`
- content of `~/.config/opensnitch/settings.conf`
- output of `sudo journalctl -ar`
- log file: /var/log/opensnitchd.log
- log file: /tmp/gui.log

**Comment by LostPoE** [comment]
Close this. 
![image](https://github.com/evilsocket/opensnitch/assets/3904973/9022a3d3-3b40-44d9-8261-f60cfb69fa53)

**Reply by LostPoE** [reply]
systemd got me. sorry. everything works now.
